caffe2/core/predictor.h

```diff
@@ -26,7 +26,8 @@ class Predictor {
       const NetDef& init_net,
       const NetDef& run_net,
       Workspace* parent = nullptr,
-      bool run_init = true);
+      bool run_init = true,
+      bool optimize = true);
```

Comment:
Why is it boolean, not integer? With this interface, optimization level is hidden from `Predictor` users.

Reply:
ah good point, I will change it